engine/env/spark/src/main/java/org/datacleaner/spark/SparkAnalysisRunner.java

```diff
@@ -54,9 +54,23 @@
     private final SparkJobContext _sparkJobContext;
     private final JavaSparkContext _sparkContext;
 
+    private final Integer _minPartitions;
+
     public SparkAnalysisRunner(JavaSparkContext sparkContext, SparkJobContext sparkJobContext) {
+        this(sparkContext, sparkJobContext, null);
+    }
+
+    public SparkAnalysisRunner(JavaSparkContext sparkContext, SparkJobContext sparkJobContext, Integer minPartitions) {
         _sparkContext = sparkContext;
         _sparkJobContext = sparkJobContext;
+        if ((minPartitions != null) && (minPartitions > 0)) {
+            _minPartitions = minPartitions;
+        } else {
+            logger.warn(
```

Comment:
True. The initial plan was to warn only if minPartitions parameter was specified, but an incorrect value. Will fix it.